examples/graphbolt/node_classification.py

```diff
@@ -363,7 +367,14 @@ def parse_args():
         "--dataset",
         type=str,
         default="ogbn-products",
-        choices=["ogbn-arxiv", "ogbn-products", "ogbn-papers100M"],
+        choices=[
+            "ogbn-arxiv",
+            "ogbn-products",
+            "ogbn-papers100M",
+            "igb-homogeneous-tiny",
+            "igb-homogeneous-small",
+            "igb-homogeneous-medium",
```

Comment:
Do you think it is better to follow the convention in the IGB paper?
```suggestion
            "igb-hom-tiny",
            "igb-hom-small",
            "igb-hom-medium",
```